probe-rs/src/debug/stack_frame.rs

```diff
@@ -54,21 +54,7 @@ impl std::fmt::Display for StackFrame {
         // Header info for the StackFrame
         writeln!(f, "Frame: {}", self.function_name)?;
         if let Some(si) = &self.source_location {
-            let separator = match &si.directory {
-                Some(path) if path.is_windows() => '\\',
-                _ => '/',
-            };
-
-            write!(
-                f,
-                "\t{}{}{}",
-                si.directory
-                    .as_ref()
-                    .map(|p| p.to_string_lossy())
-                    .unwrap_or_else(|| std::borrow::Cow::from("<unknown dir>")),
-                separator,
-                si.file.as_ref().unwrap_or(&"<unknown file>".to_owned())
-            )?;
+            write!(f, "\t{}", si.path.to_string_lossy())?;
```

Comment:
What I always wonder is how you do this properly without using `to_lossy` ...

Reply:
There is also `display()`, on `TypedPath`. But there is no way around converting something.

Reply:
Fair :) I just kind of wonder if this choice kind of ever could go bad.

Reply:
What should go bad? It will always work, just non-utf8 characters are placed by a replacement char.

There is not really an alternative, under the assumption that we can only print UTF-8.